migrations/src/main/scala/za/co/absa/enceladus/migrations/framework/MigrationUtils.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.migrations.framework
+
+object MigrationUtils {
+
+  /**
+    * Gets versioned collection name given a collection name.
+    *
+    * A collection name can be 'schema' or 'dataset', for example. The corresponding versioned collections could
+    * be 'schema_v5' or 'dataset_v5'.
```

Comment:
I am thinking about this and each migration creates a new set of collections with a coresponding number. Why not create a database with the number and keep the names the same, so it is just "plug and play" in our properties files.

Reply:
The idea behind numbered collection names is that we might have no permissions for creating a db in a preprod/prod instance, but we always have permissions to create a collection in our db. But I might be too cautious.
Lets' discuss this with others.

Reply:
Actually, that makes a lot of sense.